test/functionalTest/cases/0392_service_path_update_and_query/service_path_http_header.test

```diff
@@ -68,7 +68,7 @@ echo "+++++ 5: four service paths"
 orionCurl --url /v1/queryContext --payload "$payload" --servicePath '/home/kz/1, /home/kz/2, /home/kz/3, /home/kz/4'
 echo
 echo "+++++ 6: service paths from log file"
-grep servicePathSplit /tmp/contextBroker.log | awk -F\| '{ print $6 }' | awk -F: '{ printf "%s:%s\n", $2, $3 }'
+grep servicePathSplit /tmp/contextBroker.log | awk -F\| '{ print $6 }' | awk -F: '{ printf "%s:%s\n", $2, $3 }' | grep -v '/#'
```

Comment:
I would suggest to add an explanation about the grep, something like this (above the grep line):

````
# grep line is needed to make this test work under valgrind, as the /# is adding noise in the case of GET /version operations used to check CB in valgrind suite
```

or a better wording :)